src/style-sheets/components/_page-header.scss

```diff
@@ -0,0 +1,82 @@
+.page-header-container {
+    width: 90%;
```

Comment:
Kinda wondering if this width is correct since each of the landing view and contact us view has a different width and different amount of white space to the left and also from the top, shouldn't we keep them all consistent?

Reply:
Yeah I agree, I was wondering that too. The designs being made keep changing, I was going to base it off the footer update we have. I can update it for now but before the release of v1 we need to look at the spacing and make sure it's consistent

Reply:
Updated the width